bcipy/main.py

```diff
@@ -22,6 +22,31 @@
 log = logging.getLogger(__name__)
 
 
+def preconditions_ok(parameters) -> bool:
```

Comment:
What do you think of moving this to `helpers/validate.py`. We use a similar pattern for validating experiments. Another option for the method name: `validate_bcipy_session` or similar to be more explicit